gen/templates/datasource.md.tmpl

```diff
@@ -7,12 +7,12 @@ layout: "aci"
 page_title: "ACI: aci_{{ .ResourceName}}"
 sidebar_current: "docs-aci-data-source-aci_{{ .ResourceName}}"
 description: |-
-  Data source for {{.ResourceNameAsDescription}}
+  Data source for ACI {{.ResourceNameAsDescription}}
 ---
 
 # aci_{{ .ResourceName}} #
 
-Data source for {{.ResourceNameAsDescription}}
+Data source for ACI {{.ResourceNameAsDescription}}
```

Comment:
Do we really need to specify "ACI" here?

Reply:
The "ACI" was used in the legacy data-source doc files. To be consistent, I added that here.